frontend/css/scaffolding.less

```diff
@@ -0,0 +1,132 @@
+html {
+  font-size: 10px;
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+}
+
+body {
+  font-family: Sintony, sans-serif;
+  font-size: 14px;
+  line-height: 1.42857143;
+  color: #46433a;
+  background-color: #fff;
+}
+
+a {
+  color: #353070;
```

Comment:
Be good to use variables for all the colors in this page, if we have them somewhere as variables.
And if we don;t have a variable but they are copied in other places, let's create a variable for it

Reply:
Okay. BTW This is a temporary file which I've added right now. Eventually all these configurations needs to be moved to lobes